### PR TITLE
[MCHANGES-453] Align report output and goals names

### PR DESCRIPTION
**Pull request by slawekjaranowski**
https://issues.apache.org/jira/browse/MCHANGES-453

### CODE REVIEW

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -89,6 +87,24 @@ ${project.name}
 
   * tag <<<action/fixedIssues>>> - was changed to an attribute of <<action>> tag, it also can be comma separated.
 
+** Reports goals and output names
```
Comment (elharo) [inline src/site/apt/index.apt.vm:90]: Report goals

**Review by elharo — APPROVED** [review]

**Review by michael-o — COMMENTED** [review]
One could consider to rename the external reports to: `{name}-changes` so they are consistent with the internal (included) approach. WDYT?

**Comment by slawekjaranowski** [comment]
So will be ...

```
changes - internal from xml
jira-changes
github-changes
...
```

looks ok

**Comment by slawekjaranowski** [comment]
@michael-o - done

**Review by elharo — APPROVED** [review]

**Comment by michael-o** [comment]
Maybe the Java classes should be alike, e.g., ChangesReport.java, JiraChangesReport.java, etc?

**Comment by slawekjaranowski** [comment]
ok, next round of review

I will add a deprecated report i next PR, I would like to preserve renaming file in git history

**Comment by slawekjaranowski** [comment]
@michael-o I hope is ready

**Comment by jira-importer** [comment]
Resolve #383

**Pull request merged**